core/src/Kokkos_MathematicalFunctions.hpp

```diff
@@ -99,6 +101,16 @@ namespace Experimental {
 #endif
 #endif
 
+#if defined(KOKKOS_ENABLE_DEPRECATED_CODE_3)
+#define KOKKOS_IMPL_MATH_FUNCTIONS_DEFINED_IF_DEPRECATE_CODE_ENABLED( \
```

Comment:
```suggestion
#define KOKKOS_IMPL_MATH_FUNCTIONS_DEFINED_IF_DEPRECATED_CODE_ENABLED( \
```

Reply:
You are right.  It is a typo.  Will fix.

Reply:
Rebased and applied your suggestion